doc/release/2.8.1.rst

```diff
@@ -0,0 +1,347 @@
+Tarantool 2.8.1
+===============
+
+Released on 2021-04-21.
+
+*   Release :tarantool-release:`2.8.1`.
+*   Tagged ``2.8.1-0-ge2a1ec0``
+
+Overview
+--------
+
+2.8.1 is the :doc:`beta </release/policy>`
+version of the 2.8 release series.
+
+This release introduces 28 new features and resolves 31 bug since version :doc:`2.7.2 </release/2.7.2>`.
+There can be bugs in less common areas.
+If you find any, feel free to `report an
+issue <https://github.com/tarantool/tarantool/issues>`__ on GitHub.
+
+Notable changes are:
+
+-  Tarantool is now able to set multiple iproto threads.
+-  Setting ``box.cfg`` options with environment variables.
+-  The new ``box.ctl.promote()`` function and the concept of manual elections.
+-  Enhancements in the Lua memory profiler.
+
+Compatibility
+-------------
+
+Tarantool 2.x is backward compatible with Tarantool 1.10.x in the binary
+data layout, client-server protocol, and replication protocol.
+
+Please
+`upgrade <https://www.tarantool.io/en/doc/latest/book/admin/upgrades/>`__
+using the ``box.schema.upgrade()`` procedure to unlock all the new
+features of the 2.x series.
+
+
+Functionality added or changed
+------------------------------
+
+Core
+~~~~
+
+-   The ``exclude_null`` option can now be used in the index part definition.
+    With this option, the index filters and doesn't store tuples with "null" value
```

Comment:
Is the comma here intentional? Looks unusual, but I don't know for sure, whether it is incorrect.